frontend/src/handlers/handleResetUsername.js

```diff
@@ -0,0 +1,39 @@
+
+import axios from 'axios';
+
+const handleSubmit = async (e, email, username, newUsername, message) => {
+    e.preventDefault();
+    setMessage('');
+    if (!email || !username || !newUsername) {
+        setMessage('fill in all required information')
+        return;
+    }
+    const usernameExists = await checkUsername();
+    if (usernameExists) {
+        setMessage("username already exists")
+        return;
+    }
+    try {
+        const res = await axios.put(`http://localhost:8080/reset-username-form/update-user/`, {
```

Comment:
nit. i think you have proxy set up already so you don't need to specify the localhost:8080